docs/detect/index.rst

```diff
@@ -21,6 +21,9 @@ into source catalogs.
 Getting Started
 ===============
 
+.. image:: fermi_ts_map.png
```

Comment:
Add one sentence before or a Figure caption, that this image is the glorious result of the code example discussed below?